packit-service/packit.dev
# Google analytics

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This pull request adds support of Google analytics into our `packit.dev` page.
For a review you can ignore docs/* which are generated but important files are:

- config.toml
- layouts/partials/header.html

I have followed these articles:

- https://gohugo.io/templates/internal/
- http://cloudywithachanceofdevops.com/posts/2018/05/17/setting-up-google-analytics-on-hugo/

How to test it?

1. make generate && make server
1. visit page `localhost:1313`
1. look here `https://analytics.google.com/analytics/web/?authuser=0#/realtime/rt-overview/a143669536w205076857p198314150/` and you should see that page was accessed from your local machine.
1. Active users on site should be `1`

Hopefully you are fine with that.
